### ISSUE TITLE
 problem of resolving dependencies for CERMINE-Impl project

### ISSUE THREAD

**Issue by Tsyulrich** [CeON/CERMINE#105]
Hello everybody
please can i need your help.
I am newly with this wonderful application.

When i try to use CERMINE, as a standalone application i always have this error

Failed to execute goal on project cermine-impl: Could not resolve dependencies for project pl.edu.icm.cermine:cermine-impl:jar:1.14-SNAPSHOT: Failed to collect dependencies at edu.umass.cs.mallet:mallet:jar:0.1.3: Failed to read artifact descriptor for edu.umass.cs.mallet:mallet:jar:0.1.3: Could not transfer artifact edu.umass.cs.mallet:mallet:pom:0.1.3 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [google-maven-repository (http://google-maven-repository.googlecode.com/svn/repository/, default, releases+snapshots), yadda (http://maven.icm.edu.pl/artifactory/repo, default, releases+snapshots)]

please what's wrong?

**Comment by resourcesunite** [comment]
I had the same problem this morning. It turns out that mallet uses http but recent versions of maven only work with https. I got around it by reverting to version 3.6.3 of maven. I will contact the mallet devs and suggest they change to https.
I got the answer from Stack Overflow question "Maven stuck downloading maven-default-http-blocker".

**Comment by Tsyulrich** [comment]
Thank you so much
Please does it work fine when you use version 3.6.3 of maven?

**Comment by resourcesunite** [comment]
Yes. I had no more maven problems and I am now debugging cermine. All my problems are now with trying to understand the cermine code! Also, how to debug a Java project as I haven't worked in Java for years.
After downloading maven 3.6.3, don't forget to change your MAVEN_HOME environment variable to the location of the older build.

**Comment by bwakkie** [comment]
Just apply patch https://github.com/CeON/CERMINE/pull/91 and you are good to go with the newer more secure maven

**Comment by bcrment** [comment]
Replace the line http://maven.icm.edu.pl/artifactory/repo to https://maven.ceon.pl/artifactory/repo in CERMINE/pom.xml
